src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int height;
+    private final int sideA;
+    private final int sideB;
+
+    public IsoscelesTrapezoid(Color color, int height, int sideA, int sideB) {
+        super(color);
+        this.height = height;
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (height * 0.5 * sideA * sideB);
```

Comment:
I am not sure about the formula, please double-check

Reply:
Area = (sum of parallel sides ÷ 2) × height.
changed formula